src/main/kotlin/org/malachite/estella/task/api/TaskController.kt

```diff
@@ -33,15 +33,16 @@ class TaskController(
     @CrossOrigin
     @Transactional
     @GetMapping("/{taskId}")
-    fun getTaskById(@PathVariable taskId: Int): ResponseEntity<TaskDto> = taskService.getTaskById(taskId)
-        .let { ResponseEntity.ok(it) }
+    fun getTaskById(@PathVariable taskId: Int) = ResponseEntity(Message("Not Implemented"), HttpStatus.NOT_IMPLEMENTED) /*taskService.getTaskById(taskId)
```

Comment:
Czysto teoretycznie moglibyśmy to dodać do naszych OwnResponses XD

Reply:
Nie, bo nie chcemy tego mieć na zawsze. Jakoś w następnym sprincie powinniśmy to wyrzucić całkowicie

Reply:
Znaczy chodzi mi tak pod względem dodawania featerów to posiadanie w OwnResponses NOT_IMPLEMENTED mogłoby być spoko ale to najwyżej się doda w przyszłości bo juz zmergowane XD